src/plugins/discover/server/ui_settings.ts

```diff
@@ -47,6 +48,17 @@ import {
 } from '../common';
 
 export const uiSettings: Record<string, UiSettingsParams> = {
+  [DISCOVER_LEGACY_TOGGLE]: {
+    name: i18n.translate('discover.advancedSettings.legacyToggleTitle', {
+      defaultMessage: 'Disable new discover app',
```

Comment:
nit: should we change this to be `Disable legacy discover app`? Or is there an action item to change the logic above so that it will route to the new discover app on true?